patterns/byod.md

```diff
@@ -0,0 +1,579 @@
+
+`Azure AD Conditional Access > Policies`
+
+| Item                                            | Value                                                        |
+| ----------------------------------------------- | ------------------------------------------------------------ |
+| Existing policy name: GRANT - iOS Device access | This policy requires iOS device state to be compliant which requires enrollment. The update required to this policy is the exclude the MAM users group.<br /><br />Assignments > Exclude > rol-Agency-MAMUsers |
+
+The following is a new Conditional Access Policy to allow MAM access for BYOD option 2.
+
+| Item                                     | Value                                                        |
+| ---------------------------------------- | ------------------------------------------------------------ |
+| New policy name: GRANT - MAM BYOD access | Authorised applications on iOS and Android devices  without MDM enrollment (MAM).<br /> |
+
+* Name: `GRANT – MAM BYOD access`
+* Assignments
+  * Users and groups
+    * Include: `rol-Agency-MAMUsers`
+    * Exclude
+      * All guest and external users: `False`
+      * Directory roles: `False`
+      * Users and groups: `True`
+        * Excluded users and groups: `grp-Conditional_Access_Exclude`
+  * Cloud apps or actions
+    * Select what policy this applies to: `Cloud apps`
+    * Include: `Select apps`
+      * `Office 365`
+    * Exclude: None
+  * Conditions
+    * User risk:
+      * Configure: `No`
+    * Sign-in risk
+      * Configure: `No`
+    * Device platforms:
+      * Configure: `Yes`
+      * Include: `Select device platforms`
+        * `iOS`
+        * `Android`
+      * Exclude: 
+        * `Windows Phone`
+        * `Windows`
+        * `macOS`
+        * `Linux`
+    * Locations:
+      * Configure: `No`
+    * Client apps
+      * Configure: `Yes`
+      * Browser: `Yes`
+      * Mobile apps and desktop clients: `True`
+      * Exchange ActiveSync clients: `False`
+      * Other clients: `False`
+    * Device state:
+      * Configure: `No`
+* Access controls
+  * Grant
+    * Block access: `No`
+    * Grant access: `Yes`
+    * Require multi-factor authentication: `True`
+    * Require device to be marked as compliant: `False`
+    * Require Hybrid Azure AD joined device: `False`
+    * Require approved client app: `True`
+    * Require app protection policy: `True`
+    * Require password change: `False`
+    * Terms of Use: `False`
+    * Require all the selected controls: `Yes`
+    * Require one of the selected controls: `No`
+* Session
+  * Use app enforced restrictions: `True`
+  * Use Conditional Access App Control: `True > Custom policy`
+  * Sign-in frequency: `8 Hours`
+  * Persistent browser session: `False`
+* Enable policy: `On`
+
+The following is a new Conditional Access Policy to allow block unapproved apps for MAM users on iOS and Android.
```

Comment:
`to allow block unapproved apps` I think this is meant to be `to block unapproved apps`?